{
  "schema_version": "1.5.0",
  "id": "CVE-2013-7338",
  "modified": "2017-06-30T16:57:01",
  "published": "2014-04-22T14:00:00",
  "summary": "zipfile DoS using invalid file size",
  "details": "Python before 3.3.4 RC1 allows remote attackers to cause a denial of service (infinite loop and CPU consumption) via a file size value larger than the size of the zip file to the (1) ZipExtFile.read, (2) ZipExtFile.read(n), (3) ZipExtFile.readlines, (4) ZipFile.extract, or (5) ZipFile.extractall function.",
  "affected": [
    {
      "ranges": [
        {
          "type": "GIT",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "5ce3f10aeea711bb912e948fa5d9f63736df1327"
            }
          ],
          "repo": "https://github.com/python/cpython"
        }
      ]
    }
  ],
  "references": [],
  "database_specific": {
    "cwe_ids": []
  }
}